mono/mini/method-to-ir.c

```diff
@@ -710,7 +710,11 @@ mono_find_leave_clauses (MonoCompile *cfg, unsigned char *ip, unsigned char *tar
 		clause = &header->clauses [i];
 		if (MONO_OFFSET_IN_CLAUSE (clause, (ip - header->code)) && 
 		    (!MONO_OFFSET_IN_CLAUSE (clause, (target - header->code)))) {
-			res = g_list_append_mempool (cfg->mempool, res, clause);
+			MonoLeaveClause *leave = mono_mempool_alloc (cfg->mempool, sizeof (MonoLeaveClause));
+			leave->index = i;
```

Comment:
Suggest using alloc0 for defensive coding.

Reply:
Strongly agreed, zeroing is great, and allows for evolution of types without visiting every initializer. It is the next best thing to C++ constructors, or perhaps better due to efficiency and works with globals. memset, ZeroMemory, HEAP_ZERO_MEMORY, calloc, etc.